torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1885,9 +1885,40 @@ RegisterOperators reg2({
           push(stack, a / b);
           return 0;
         }),
-
     Operator(
-        "aten::floor(float a) -> int",
+        "aten::fmod(float a, float b) -> float",
+        [](Stack& stack) {
+          double a, b;
+          pop(stack, a, b);
+          push(stack, std::fmod(a, b));
+          return 0;
+        }),
+    Operator(
+        "aten::modf(float a, float b) -> float",
+        [](Stack& stack) {
+          double a, b;
+          pop(stack, a, b);
+          push(stack, std::modf(a, &b));
+          return 0;
+        }),
+    Operator(
+        "aten::acos(float a) -> float",
+        [](Stack& stack) {
+          double a;
+          pop(stack, a);
+          push(stack, std::acos(a));
+          return 0;
+        }),
+    Operator(
+        "aten::remainder(float a, float b) -> float",
+        [](Stack& stack) {
+          double a, b;
+          pop(stack, a, b);
+          push(stack, std::fmod((b + std::fmod(a, b)), b));
+          return 0;
+        }),
+    Operator(
+        "aten::floor(float a) -> float",
```

Comment:
from below, looks like this line is still supposed to be -> int ?

Reply:
Thanks for pointing out. Have now fixed it to return int.

Reply:
IIRC `floor` returns a float in Python, so we want to keep that behavior instead.